docker/mem0/main.py

```
@@ -0,0 +1,329 @@
+)
+
+load_dotenv()
+
+ADMIN_API_KEY = os.environ.get("ADMIN_API_KEY", "")
+MIN_KEY_LENGTH = 16
+
+if not ADMIN_API_KEY:
+    logging.warning(
+        "ADMIN_API_KEY not set - API endpoints are UNSECURED! "
+        "Set ADMIN_API_KEY environment variable for production use."
+    )
```

Ruff (0.15.6)

[warning] `warning()` call on root logger

(LOG015)